decred_wallet/Extensions/Extensions.swift

```diff
@@ -69,3 +69,11 @@ extension Date {
         self = Date(timeIntervalSince1970: TimeInterval(milliseconds) / 1000)
     }
 }
+
+extension UIRefreshControl {
+    func programaticallyBeginRefreshing(in tableView: UITableView) {
```

Comment:
`programaticallyBeginRefreshing`? Is there any executed swift code that is not "programatically" executed? I'm quite sure you can use `beginRefreshing` as an overloaded method.

Reply:
Renamed to showLoader